src/chapters/control-flow-and-collections/studio.rst

```diff
@@ -0,0 +1,110 @@
+.. _counting-characters-studio:
+
+Studio: Counting Characters
+===========================
+
+In this studio, you will write a program to count the number of times each
+character occurs in a string and then print the results to the console.
+
+Feel free to prompt the user for a string. However, for the sake of simplicity,
+you might want to start by hard-coding some text and storing it in a variable.
+For your convenience, here is some `lorem ipsum <https://loremipsum.io/>`__ text:
+
+   Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc accumsan sem ut 
+   ligula scelerisque sollicitudin. Ut at sagittis augue. Praesent quis rhoncus justo. 
+   Aliquam erat volutpat. Donec sit amet suscipit metus, non lobortis massa. Vestibulum 
+   augue ex, dapibus ac suscipit vel, volutpat eget massa. Donec nec velit non ligula 
+   efficitur luctus.
+
+.. admonition:: Tip
+
+   Remember, you can turn a ``string`` object into an array of characters
+   using:
+
+   .. sourcecode:: c#
+
+      char[] charactersInString = myString.ToCharArray();
+
+Some Items to Ponder Before Starting
+------------------------------------
+
+#. There are multiple ways to approach this task, but one way involves the
+   following steps:
+
+   a. Loop through the string one character at a time,
+   b. Store and/or update the count for a given character using an appropriate
+      data structure.
+   c. Loop through the data structure to print the results (one character and its
+      count per line).
+
+#. Which type of data structure (``List``, ``array``, or ``Dictionary``)
+   should you use to store character counts? Any can be made to work, but there
+   is a BEST choice.
```

Comment:
I might rephrase to "Any of these options would work, but using one of these data structures will optimize your code's efficiency." You could really use any effect, but the phrase "there is a BEST choice" reads a little awkward to me.